### PR TITLE
Add Song

### PR DESCRIPTION
**Pull request by ghost**
Added Don't Stop Me Now by Queen to the song list. :bloblul:

### CODE REVIEW

File: assets/js/app.min.js [hunk 1]
```diff
@@ -1 +1 @@
...
...ser),query.hasOwnProperty("visualizer")&&setTimeout(function(){visualizer.toggled=!0,renderVisualizer()},2e3)}config.audio&&r(function(){function e(){var n=config.audioFiles[Math.floor(Math.random()*config.audioFiles.length)];return i===n?e():(Logger.info("[Player] Changing song..."),void t(n))}function t(t){i=t,o.innerText=t[0],o.setAttribute("href",t[1]),n.setAttribute("src","assets/music/"+t[0]+".mp3"),n.load(),n.play(),n.addEventListener("ended",e),Logger.info("[Player] Started playing "+t[0]+".")}var n=document.getElementById("audio"),o=document.getElementById("song"),i=null;return n.volume=config.audioVolumeDefault,document.getElementById("skip-song").onclick=e,Mousetrap.bind("s",e),Mousetrap.bind("up",function(e){e.preventDefault(),n.volume=1==n.volume?n.volume:Math.round(10*(n.volume+.1))/10,Logger.info("[Player] Volume incremented: "+n.volume)}),Mousetrap.bind("down",function(e){e.preventDefault(),n.volume=0==n.volume?n.volume:Math.round(10*(n.volume-.1))/10,Logger.info("[Player] Volume decremented: "+n.volume)}),query.hasOwnProperty("song")&&config.audioFiles[query.song]?(Logger.info("[Player] Playing song from query string."),t(config.audioFiles[query.song])):void e()});
```
Comment (Dougley) [inline assets/js/app.min.js]: This is autogenerated when compiling, you don't need to update this manually.

Comment (ghost) [reply assets/js/app.min.js]: Cool, should I remove it from this pr?

Comment (Dougley) [reply assets/js/app.min.js]: 👍

**Comment by ghost** [comment]
image

**Comment by ghost** [comment]
either i'm confused or github is confused. i restored the file and committed again, but it still views the file as changed

**Comment by ghost** [comment]
Status?

**Pull request merged**